### ISSUE TITLE
Questions about training on the pedestrian dataset

### ISSUE THREAD

**Issue by Su12234747** [ultralytics/ultralytics#1901]
### Search before asking

- [X] I have searched the YOLOv8 [issues](https://github.com/ultralytics/ultralytics/issues) and [discussions](https://github.com/ultralytics/ultralytics/discussions) and found no similar questions.

### Question

Hi!!!   I want to use yolov8m's framework and pre-trained weights to train on the CrowdHuman dataset for the pedestrian category, should I:
1. Convert the category of the CrowdHuman dataset to the coco dataset (the pedestrian category number in the coco dataset is 0, but the pedestrian category number in the CrowdHuman dataset is 1); and delete other categories in the CrowdHuman dataset;
2. Change the labeling format of the CrowdHuman dataset to the labeling method of the Anchor-Free format
I have these two questions, thank you~~. In addition, do you have any suggestions for pedestrian training, thank you again! Looking forward to your reply!

**Comment by glenn-jocher** [comment]
Hello @Su12234747! Thank you for your interest in YOLOv8! To answer your first question, it may be easiest to follow the format of the COCO dataset by labelling your pedestrian class with category 0. However, this is not strictly necessary. Alternatively, you could adjust the configuration of `yolov8m.yaml` to match the label format of the CrowdHuman dataset, such as by modifying the number of categories in `nc:`. Nonetheless, this depends on how comfortable you are with changing the existing standard for the dataset you are using. 

Regarding your second question, YOLOv8 is capable of training both anchor-based and anchor-free object detection models. You can refer to the [docs](https://docs.ultralytics.com) to find more information on this topic.

For training on pedestrian detection we recommend the following guidance:

- Use smaller image sizes during training (e.g. `size: 416`), as pedestrians are generally smaller objects relative to their surroundings.
- Start with pre-trained weights (e.g. [yolov8-s](https://github.com/ultralytics/yolov5/tree/master/models/yolov8)) on random subsets of your training data to speed up convergence.
- Adjust the weights of the `obj` and `cls` loss functions to emphasize object detection, which may require some experimentation.
- Ensure that you have a balanced sample of pedestrian images and non-pedestrian images in your training data to avoid bias towards the pedestrian class.

I hope this helps! Don't hesitate to let us know if you have any additional questions or if we can clarify anything further for you.
